gpu/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# vars file for gpu
+home_dir: "/home/ubuntu"
```

Comment:
Not sure all Ubuntu installations have this user/directory. Let's rather default to "/root" directory. Ideally, it should be a temporary working directory, e.g. `/root/ansible.<RAND>/` or so. I think ansible might have a built-in framework for that.